### PR TITLE
feat(icons): add no-fees icon

### PR DESCRIPTION
**Pull request by apalski**
## Description

Add a new icon, ic_no-fees.svg, to assets for the new QVP "No booking fees"
https://app.clubhouse.io/qantasaccommodation/story/73287/as-a-qantas-hotels-marketer-i-d-like-to-update-the-qvp-across-the-hotels-website

## Notes
- I removed the Cypress fakeServer from assets/index.js as it was not being used

💁‍

## Looks like this

Screen Shot 2021-07-12 at 4 46 57 pm

🤨

---

**PR check-list**

🚨 **IMPORTANT:** Please review the [guidelines for contributing](./CONTRIBUTING.md) to this repository first! 👀

- [ ] You've requested a review from at least ONE person. Preferably a roo-ui maintainer ([Angus](https://github.com/angusfretwell), [Philip](https://github.com/philipwindeyer), [Eric](https://github.com/eneo5541), [Mike](https://github.com/memcc), or [Long](https://github.com/KieraDOG)).
  - Note: If your PR has not been reviewed within two days of requesting a review, feel free to seek reviews from other [team members of Qantas Hotels](https://github.com/orgs/hooroo/people).
- [ ] Your final merge commit message is in the format `<type>(<scope>): <description>` as per [the contributing guide](https://github.com/hooroo/roo-ui/blob/master/.github/CONTRIBUTING.md#commit-naming)
  - Example: `fix(dependencies): minor version bump to resolve security vulnerability in set-value`

### CODE REVIEW

**Comment by ehyland** [comment]
:construction_worker: Deploy Preview for *roo-ui* processing.


:hammer: Explore the source changes: 44eebb427ae74d4ad3c2a560eb123c600aff150d

:mag: Inspect the deploy log: [https://app.netlify.com/sites/roo-ui/deploys/60ecf152ef26d90008d44754](https://app.netlify.com/sites/roo-ui/deploys/60ecf152ef26d90008d44754?utm_source=github&utm_campaign=bot_dl)

**Review by philipwindeyer — APPROVED** [review]
👏 🚀

File: src/assets/index.js [hunk 1]
```diff
@@ -18,6 +18,7 @@ export { default as headset } from './headset.svg';
 export { default as find } from './find.svg';
 export { default as pointsPlusPay } from './points-plus-pay.svg';
 export { default as points } from './points.svg';
+export { default as icNoFees } from './ic_no-fees.svg';
```
Comment (darmaine-qh) [inline src/assets/index.js:21]: suggestion: Can we please add naming that's more descriptive of the icon? Maybe we can also ask Kym what she wants to name it?

This applies to the svg filename too, i.e. `no-fees.svg` and then we can name the new icon, `noFees`

**Review by darmaine-qh — CHANGES_REQUESTED** [review]
Looking good Annette!
One thing, if we intend to squash and merge the PR, we'll have to change the title to include the commit prefix, i.e. `feat: add no fees icon`. This is so that semantic-release can pick it up and bump up the versioning etc 😄 

And just a minor change on naming.

File: src/assets/index.js (hunk 1 above)
Comment (apalski) [reply src/assets/index.js:21]: That makes sense thanks Darmaine 😄  I will check with Kym as well

**Review by darmaine-qh — APPROVED** [review]
🎉 🚀 

Well done! Let's ship it!

**Pull request merged**